src/main/scala/com/cognite/sdk/scala/v1/resources/sequenceRows.scala

```diff
@@ -18,17 +18,17 @@ class SequenceRows[F[_]](val requestSession: RequestSession[F])
   override val baseUri = uri"${requestSession.baseUri}/sequences/data"
 
   implicit val errorOrItemsSequenceRowsResponseDecoder
-      : Decoder[Either[CdpApiError, Items[SequenceRowsResponse]]] =
-    EitherDecoder.eitherDecoder[CdpApiError, Items[SequenceRowsResponse]]
+      : Decoder[Either[CdpApiError, SequenceRowsResponse]] =
+    EitherDecoder.eitherDecoder[CdpApiError, SequenceRowsResponse]
   implicit val errorOrUnitDecoder: Decoder[Either[CdpApiError, Unit]] =
     EitherDecoder.eitherDecoder[CdpApiError, Unit]
 
-  def insertById(id: Long, columns: Seq[Long], rows: Seq[SequenceRow]): F[Unit] =
+  def insertById(id: Long, columnExternalIds: Seq[String], rows: Seq[SequenceRow]): F[Unit] =
```

Comment:
They changed the API so those are now external ids instead of internal (column) ids, so I thought this would be clearer. However, since the API calls it "columns" still it's better to stick to that here as well.